engines/llama/build.gradle

```diff
@@ -43,15 +44,15 @@ processResources {
         }
         copy {
             from jnilibDir
```

Comment:
Did you verify whether changes to the files in this folder will trigger task execution?

Reply:
The files are versioned and downloaded from a remote URL given the input property versions.

My reasoning is since the files are versioned (and the processing is always the same), then if the input property versions do not change, the files will not either. (In fact I cannot see a way how to test this without modifying the task itself)

Reply:
Hm, that's true, now that the version is tracked that should take care of it 👍